lib/Botkit.js

```diff
@@ -12,4 +13,5 @@ module.exports = {
     twilioipmbot: TwilioIPMbot,
     botframeworkbot: BotFrameworkBot,
     consolebot: ConsoleBot,
+    nexmobot: NexmoBot
```

Comment:
Since there are plans to offer IP Messaging/Conversations through the Nexmo service it may be worth explicitly naming this `NexmoSMS` bot if it is only going to manage SMS messaging.